proto/redpanda/core/admin/v2/shadow_link.proto

```diff
@@ -102,6 +102,9 @@ message DeleteShadowLinkRequest {
             type: "redpanda.core.admin.ShadowLinkService/ShadowLink"
         }
     ];
+    // If true, then any active shadow topics will be failed over first before
+    // the link is deleted
```

Comment:
In my mind force seems to address a different problem

Failover (i.e., opening the Kafka API) occurs regardless of whether force=true or force=false is used.

* When force=false (default), guardrails are in place to prevent the deletion of links that have active mirroring topics. This is designed to protect against accidental deletion by users.

* When force=true, the intent is explicit: proceed with deletion regardless of any active mirroring topics. In this case, the link is deleted, and the topics become active.